lms/migrations/versions/7e4124035651_add_the_provisioning_column.py

```diff
@@ -0,0 +1,31 @@
+"""
+Add the provisioning column.
+
+Revision ID: 7e4124035651
+Revises: efcf8671f4d3
+Create Date: 2018-12-07 11:38:55.717986
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "7e4124035651"
+down_revision = "efcf8671f4d3"
+
+
+def upgrade():
+    op.add_column(
+        "application_instances",
+        sa.Column(
+            "provisioning",
```

Comment:
Really don't want to throw a wrench in the works with all the dependent PRs stacked up, and it's not vital, but I feel like a clearer name like `enable_provisioning` could be helpful. But it's longer, and changing it would probably trip us up a bunch at this point.